components/log-viewer-webui/server/src/plugins/MongoSocketIoServer/index.ts

```
@@ -0,0 +1,392 @@
+        if ("undefined" === typeof subscribedQueryIds) {
+            return;
+        }
+
+        for (const queryId of subscribedQueryIds) {
+            this.#unsubscribe(socket, queryId);
+        }
+
+        this.#subscribedQueryIdsMap.delete(socket.id);
+        this.#fastify.log.debug(
+            "Subscribed queryIDs map" +
+            ` ${JSON.stringify(Array.from(this.#subscribedQueryIdsMap.entries()))}`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)